src/v2i-hub/CARMAStreetsPlugin/manifest.json

```diff
@@ -46,6 +46,11 @@
             "default": "v2xhub_scheduling_plan_sub",
             "description": "Apache Kafka topic plugin will transmit message to."
         },
+        {
+            "key": "subscribeToSpatTopic",
+            "default": "spat_json",
```

Comment:
@dan-du-car Saeid created a topic name as well which he called "modified_spat". I don't care which one we use but they should be the same.